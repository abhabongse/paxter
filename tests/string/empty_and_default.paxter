@python##"
    left = "ABC"
    right = "XYZ"
"##\
@left@![and]@right
login@@example.com
