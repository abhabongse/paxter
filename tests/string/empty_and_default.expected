ABC[and]XYZ
login@example.com
